config/site_settings.yml

```diff
@@ -1985,7 +1985,15 @@ developer:
   enable_safe_mode:
     default: true
     client: true
-  enable_experimental_sidebar:
+  enable_experimental_sidebar_hamburger:
+    default: false
+    client: true
+    hidden: true
+  enable_sidebar:
+    default: false
+    client: true
+    hidden: true
+  enable_per_user_sidebar_configuration:
```

Comment:
I don't see this new setting being used anywhere - are we going to be using it later?

Reply:
oops I forgot to remove this. It is no longer required.